### ISSUE TITLE
Remediated Reporter Letters for onrr.gov

### ISSUE THREAD

**Issue by cmathers1** [ONRR/onrr.gov-site#1149]
Good morning, please see attached remediated Dear Reporter Letters for onrr.gov
[8-27-12--1.pdf](https://github.com/ONRR/onrr.gov-site/files/7467463/8-27-12--1.pdf)
[880204.pdf](https://github.com/ONRR/onrr.gov-site/files/7467466/880204.pdf)
[880311.pdf](https://github.com/ONRR/onrr.gov-site/files/7467467/880311.pdf)
[20001207.pdf](https://github.com/ONRR/onrr.gov-site/files/7467468/20001207.pdf)
[20040922.pdf](https://github.com/ONRR/onrr.gov-site/files/7467469/20040922.pdf)
[20071213.pdf](https://github.com/ONRR/onrr.gov-site/files/7467475/20071213.pdf)
[20081028.pdf](https://github.com/ONRR/onrr.gov-site/files/7467476/20081028.pdf)
[20091007.pdf](https://github.com/ONRR/onrr.gov-site/files/7467488/20091007.pdf)
[20111205.pdf](https://github.com/ONRR/onrr.gov-site/files/7467489/20111205.pdf)

**Comment by Maroyafaied** [comment]
@cvojslavek let me know if you have time to take this issue. @cthomasONRR can you review after the changes are in dev?

**Comment by cthomasONRR** [comment]
Sure! @cvojslavek please let me know if you need any help, or don't have time today!

**Comment by cvojslavek** [comment]
@Maroyafaied I can take this issue today. @cthomasONRR thank you, I'll let you know when the changes are ready for your review in dev before going live.

**Comment by cvojslavek** [comment]
@cthomasONRR ready for your review at http://2017onrr/DearRep.htm, thank you. 

For the 7/9 accessible documents:
- [x] 8-27-12--1.pdf
- [x] 880311.pdf
- [x] 20001207.pdf
- [x] 20071213.pdf
- [x] 20081028.pdf
- [x] 20091007.pdf
- [x] 20111205.pdf

I followed the steps below:
- [x] Recheck docs for accessibility
- [x] Upload docs in dev site
- [x] Review docs in dev site

After your review, I’ll do the following for the 7/9 accessible documents:
- [ ] Publish to the live site
- [ ] Update Accessibility tracking sheet
- [ ] Check the live site

These two are showing accessibility issues:
- [ ] 880204.pdf
- [ ] 20040922.pdf

![image.png](https://images.zenhubusercontent.com/612e37e94df0eb7eee8b9988/173a5cd5-d1b5-4ebb-b489-667f547abf10)
![image.png](https://images.zenhubusercontent.com/612e37e94df0eb7eee8b9988/a36c77e6-3f56-40c4-b7fe-c20ca79a33b7)

**Comment by cthomasONRR** [comment]
@cvojslavek Everything for those 7 documents looks good to me.

@cmathers1 2 of the documents did not pass the assessibility check:
 880204.pdf - Table Header and Table Summary failed.
 20040922.pdf - Several fails within the document
Cat has screen captures in the comment above showing the issues. Please let us know if you need help resolving the issues in these two documents. Thank you again for volunteering to do these documents!!!!

**Comment by cvojslavek** [comment]
@cthomasONRR  @cmathers1 the following 7 documents are on the live site at https://onrr.gov/DearRep.htmONRR (please note that you may have to use “Ctrl+F5” to refresh): 
- [x] 8-27-12--1.pdf
- [x] 880311.pdf
- [x] 20001207.pdf
- [x] 20071213.pdf
- [x] 20081028.pdf
- [x] 20091007.pdf
- [x] 20111205.pdf

Thank you for your review @cthomasONRR, for the 7 documents I performed the steps below:
- [x] Publish to the live site
- [x] Update Accessibility tracking sheet
- [x] Check the live site

@cthomasONRR I added 880204.pdf to the dev site, http://2017onrr/DearRep.htm, ready for your review. This one is showing table issues with my updated version of Adobe and not in prior versions of Adobe.

**Comment by cthomasONRR** [comment]
Looks good to me! We can follow up with Christina sometime tomorrow.

**Comment by cvojslavek** [comment]
@cthomasONRR @cmathers1 **880204.pdf** is on the live site at https://onrr.gov/DearRep.htmONRR (please note that you may have to use “Ctrl+F5” to refresh):

Thank you for your review @cthomasONRR, for **880204.pdf** I performed the steps below:
- [x] Publish to the live site
- [x] Update Accessibility tracking sheet
- [x] Check the live site

**Comment by cthomasONRR** [comment]
Christina said that we can close this issue- she will make make another issue for the revised document.
